tests/xharness/Jenkins.cs

```diff
@@ -3355,8 +3330,8 @@ protected override async Task RunTestAsync ()
 					proc.StartInfo.Arguments = $"--debug {reporter} {WorkingDirectory} {results}";
 
 					Jenkins.MainLog.WriteLine ("Executing {0} ({1})", TestName, Mode);
-					var log = Logs.Create ($"execute-xtro-{Timestamp}.txt", "Execution log");
-					log.WriteLine ("{0} {1}", proc.StartInfo.FileName, proc.StartInfo.Arguments);
+					var log = Logs.Create ($"execute-xtro-{Timestamp}.txt", Log.EXECUTION_LOG);
+						log.WriteLine ("{0} {1}", proc.StartInfo.FileName, proc.StartInfo.Arguments);
```

Comment:
This tab looks funny. I don't think we added one in any of the PRs.

Reply:
I’ll check, that might have been VSMac doing it’s thing...

Reply:
@mandel-macaque looks like nor me or VSMac to blame this time 🙈 see

https://github.com/xamarin/xamarin-macios/blame/5cc1406ec41db56eccd943e8c02f8d19c3eb284d/tests/xharness/Jenkins.cs#L3321